tests/integration/test_api.py

```diff
@@ -655,6 +655,26 @@ def test_get_unanswered_questions_api(client):
     assert data["status"] == "found"
 
 
+def test_get_unanswered_questions_api_with_country_filter(client):
+    ProductInsight.delete().execute()  # remove default sample
+
+    # test for filter with "country"
+
+    ProductInsightFactory(
+        type="location", value_tag="en:dates", barcode="032", countries=["en:india"]
+    )
```

Comment:
Adding only one product, you don't really test the filter is effective.

To test that you should add another product in another country
```suggestion
    ProductInsightFactory(
        type="location", value_tag="en:dates", barcode="033", countries=["en:france"]
    )
```